mne/preprocessing/maxwell.py

```diff
@@ -0,0 +1,492 @@
+# Authors: Mark Wronkiewicz <wronk.mark@gmail.com>
+#          Jussi Nurminen <jnu@iki.fi>
+
+# This code was adapted and relicensed (with BSD form) with permission from
+# Jussi Nurminen
+
+# License: BSD (3-clause)
+
+# Note, there are an absurd number of different possible notations for
+# spherical coordinates, which confounds the notation for spherical harmonics.
+# Here, we purposefully stay away from shorthand notation in both and use
+# explicit terms (like 'azimuth' and 'polar') to avoid confusion.
+
+# TODO: write in equation numbers from Samu's paper
+
+from __future__ import division
+import numpy as np
+from scipy.special import lpmv
+from scipy.linalg import pinv
+from scipy.misc import factorial as fact
```

Comment:
you will need to nest at least the `scipy.linalg` if not also the `scipy.misc` or `scipy.special` imports to save `mne` import time -- run `nosetests mne/tests/test_import_nesting.py` and you will see it